src/e2e-test/regression-test/regression-test.spec.js

```diff
@@ -0,0 +1,61 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+const supertest = require('supertest');
+const express = require('express');
+const fs = require("fs");
+const path  = require('path');
+const routes = require('../../routes');
+const constants = require('../../lib/constants/constants');
+const cases = require('./config');
+const utils = require('./util/utils');
+
+const MAX_TEST_TIME = 10000;
+const API_KEY_HEADER = "X-MS-CONVERSION-API-KEY";
+const API_KEY = '=2&jcNFjtsvp=V97mBcH%T_kU=5SMGm=';
```

Comment:
where is this key from? Is it a secret?

Reply:
No it's not a secret but a randomly generated string, I choose it just because it's more like a "key".

Reply:
Let's make it simple. It looks suspicious